pkg/tests/scrapers/scrapers_suite_test.go

```diff
@@ -0,0 +1,55 @@
+package scrapers_test
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/joho/godotenv"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+)
+
+func TestScrapers(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Scrapers Suite")
+}
+
+func TestMain(m *testing.M) {
```

Comment:
ditto here with the above - this happens because the config singleton tries to parse os.Args. And the Twitter scrapper calls the config singleton, which at the first load does parse flags from args. 

My suggestion here would be: if the tests needs such a "forced" execution or "uncommon" setup it likely means the code under test needs adaptation